httpchecker_test.go

```diff
@@ -11,6 +11,7 @@ import (
 func TestHTTPChecker(t *testing.T) {
 	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("X-Checkup", r.Header.Get("X-Checkup"))
+		w.Header().Set("X-Host", r.Host)
```

Comment:
Why set this? What is X-Host...

Reply:
Well mostly just a copy of the test made for headers, i will refactor this to be easier to understand.

Reply:
Okay. Where is it being used?

Reply:
This was to prove the host header was being set, it is a copy of the set headers test. the test that uses this is from line 122

Reply:
As I understand the tests here, we test to see if the response has the value `"www.echo.com"`, so the `X-HOST' is to ensure that the response contains the header we set so we know it has been set.  

it is a copy of line 108 to 120 
```
	hc.Headers = http.Header{
		"X-Checkup": []string{"Echo"},
	}
	hc.MustContain = "Echo"
	hc.ThresholdRTT = 0
	result, err = hc.Check()
	if err != nil {
		t.Errorf("Didn't expect an error: %v", err)
	}
	if got, want := result.Down, true; got != want {
		t.Errorf("Expected result.Down=%v, got %v", want, got)
	}
```

Reply:
Oh... okay. I believe MustContain only applies to the response body, not the response headers as well.